tests/http/cache-control.yml

```diff
@@ -43,7 +43,7 @@ mock:
         test: test
     response:
       headers:
-        Cache-Control: "max-age=7200"
+        Cache-Control: "max-age=7200, private"
```

Comment:
We're lacking tests to public cache on this file

Reply:
Oh, yes. Missed it. Updating...

Reply:
Since the cache is by default `public`, async_graphql is not setting the `public` flag. This would still adhere to the standards. Do you want me to update the behavior to show the `public` directive if it's present?

Reply:
@A-N-uraag what @ologbonowiwi meant was in this file there were two test
1. both cache control headers are public, expectation is to set the mimimum of two.
2. no-cache should in one request should convert the resultant header to no store.
you updated the first one to test private so there is not test which tests the both public scenario.

Expected tests
1. A - Public B - Public. result Min of A and B Public
2. A - Public B - Private result min of A and B Private
3. A - Public B - No cache result No cache
4. A - Private B - Nocache result No cache

Reply:
Ah, got it! Added the other scenarios. A small note @amitksingh1490: Case 4 would result in `No-Cache, Private` since No-Cache still indicates that a validation before re-use is required. But private should still be present to indicate that only private caches are allowed to store this response.